x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -78,10 +79,17 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
 
   const [isHostIsolationPanelOpen, setIsHostIsolationPanel] = useState(false);
 
-  const [isolateAction, setIsolateAction] = useState('isolateHost');
+  const [isolateAction, setIsolateAction] = useState<'isolateHost' | 'unisolateHost'>(
```

Comment:
should maybe this type reference the one defined in `EndpointIsolateSuccessProps`?

```suggestion
  const [isolateAction, setIsolateAction] = useState<EndpointIsolateSuccessProps['isolateAction']>(
```